src/main/java/bookbob/entity/Visit.java

```diff
@@ -0,0 +1,49 @@
+package bookbob.entity;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Visit {
+    private LocalDateTime visitDate;
+    private String diagnosis;
+    private List<String> medication;
+
+    public Visit(LocalDateTime visitDate, String diagnosis, List<String> medication) {
+        this.visitDate = visitDate;
+        this.diagnosis = "";
+        this.medication = new ArrayList<>();
+    }
```

Comment:
Perhaps instead of taking diagnosis as a String, can take in a list of diagnosis (similar to medications)
The List<String> medication could be changed to List<String> medications for the correct usage of plural form too

Reply:
Ok will do!